### DISCUSSION TITLE
Unable to convert PDF file to PNG image.

### DISCUSSION THREAD

**Discussion by Liujiahao0** [ImageMagick/ImageMagick#6521] [Help]
### ImageMagick version

7.1.1-13

### Operating system

Linux

### Operating system, version and so on

ubuntu-20.04

### Description

After executing the/home/ljh/ImageMagick/build/bin/convert in/1.pdf in/1.png command for dozens of hours, the PDF file still cannot be converted to a png image. 

### Steps to Reproduce

My Compilation Process：
git clone https://github.com/ImageMagick/ImageMagick.git
cd ImageMagick
CFLAGS="-g -O0" CXXFLAGS="-g -O0" ./configure --prefix="$HOME/ImageMagick/build/"
sh configure
chmod u+x configure
./configure --prefix="$HOME/ImageMagick/build/"
make
sudo make install

reproduce: /home/ljh/ImageMagick/build/bin/convert in/1.pdf in/1.png

### Images

[1.pdf](https://github.com/ImageMagick/ImageMagick/files/12184063/1.pdf)

**Comment by snibgo** [comment]
With Ghostcript v10, IM can't read `1.pdf`:
```
magick -density 1 1.pdf info:

magick: Postscript delegate failed `[ghostscript library 10.00] -q -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 "-sDEVICE=pamcmyk32" -dTextAlphaBits=4 -dGraphicsAlphaBits=4 "-r1x1" -dPrinted=false  "-sOutputFile=C:/Users/Alan/AppData/Local/Temp/magick-jDA_8nBZaMlgl8mpI7WSjaboxHUbtHls%d" "-fC:/Users/Alan/AppData/Local/Temp/magick--dT01pbWm62ohsg_IEDn2QGItlrDbhTu" "-fC:/Users/Alan/AppData/Local/Temp/magick-3dgPJw5OylywRfHoNSw2_hoif23XeFCC"': Unrecoverable error: rangecheck in setcolorscreen
Operand stack:
    0.0624961  0  --nostringval--  0.0624961  0  --nostringval--  0.0624961  0  --nostringval--  0.0624961  0  --nostringval--
 @ error/ghostscript-private.h/InvokeGhostscriptDelegate/184.
```

Adobe Acrobat Reader also reports many problems with `1.pdf`.
